src/highdicom/sr/templates.py

```diff
@@ -3889,6 +4004,16 @@ def get_planar_roi_measurement_groups(
         """  # noqa: E501
         # FIXME: reference type and referenced sop instance uid
 
+        if graphic_type is not None:
+            if not isinstance(
+                graphic_type,
+                (GraphicTypeValues, GraphicTypeValues3D)
+            ):
+                raise TypeError(
+                    'graphic_type must be of type GraphicTypeValues or '
+                    'GraphicTypeValues3D, or None.'
```

Comment:
For improved consistency with the style of other error messages:

```suggestion
                    'Argument "graphic_type" must be of type GraphicTypeValues, '
                    'GraphicTypeValues3D or None.'
```